src/vector.js

```diff
@@ -141,6 +141,13 @@
 
     toObject: function() {
       return { x: this.x, y: this.y };
+    },
+
+    rotate: function (radians) {
+      var newX = this.x * Math.cos(radians) - this.y * Math.sin(radians);
+      var newY = this.x * Math.sin(radians) + this.y * Math.cos(radians);
+      this.set(newX, newY);
```

Comment:
Method **set()** is already chainable. So we can use `return this.set(newX, newY);` right way.